src/pages/MeetingSetup/MeetingRecap.jsx

```diff
@@ -1,14 +1,29 @@
-import React from 'react';
-import { useDispatch } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import { useNavigate, useLocation } from 'react-router-dom';
-import { createMeeting } from '../../redux/slices';
+import {
+  createMeeting,
+  selectMeetings,
+  resetMeetingStatus,
+} from '../../redux/slices';
 
 export default function MeetingRecap(props) {
   const dispatch = useDispatch();
   const navigate = useNavigate();
 
+  const meetings = useSelector(selectMeetings);
+
   const location = useLocation();
 
+  useEffect(() => {
+    // on load, make sure meeting state is cleared
+    dispatch(resetMeetingStatus);
+  }, []);
+
+  useEffect(() => {
+    if (meetings.meeting.id && !meetings.error) navigate('/');
```

Comment:
what is this doing exactly? The page doesn't load for me but it looks like it is navigating away if there is a meeting and there's no error.